lib/linguist/languages.yml

```diff
@@ -1110,6 +1110,9 @@ Dockerfile:
   - ".dockerfile"
   filenames:
   - Dockerfile
+  - "dockerfile"
```

Comment:
Is the lower-case D common? I've never encountered it before.

@lildude Any way we can search for that on GitHub.com? Maybe through Google BigQuery?

Reply:
Not on GitHub itself as there's no case-sensitive search. I have no idea about BigQuery - never used it.

Reply:
> I have no idea about BigQuery - never used it.

I have now. 🎉

There are _only_ 403 instances `dockerfile` in the BigQuery dataset according to this query:

```sql
SELECT repo_name, path
FROM `bigquery-public-data.github_repos.files`
WHERE path LIKE '%/dockerfile'
GROUP BY repo_name, path
LIMIT 5000
```

There are well over 5000 examples of `Dockerfile` though 😄

Reply:
> > I have no idea about BigQuery - never used it.
> 
> I have now. :tada:

Wow! How did you do that so quickly? Is there a good tutorial for that? It would definitely improve our ability to asses in-the-wild usage.

Reply:
I searched Google, found https://codelabs.developers.google.com/codelabs/bigquery-github/index.html?index=..%2F..index#0, signed into my work GCP account (I was already signed in, but could have used my personal account too) and put my terrible SQL skills to work 😁.